src/Util/Test.php

```diff
@@ -517,7 +518,7 @@ private static function getDataFromDataProviderAnnotation($docComment, $classNam
                     $data = $dataProviderMethod->invoke($object, $methodName);
                 }
 
-                if ($data instanceof Iterator) {
+                if ($data instanceof Traversable) {
                     $data = \iterator_to_array($data);
```

Comment:
@keradus `Iterable` is not not class or interface, it is pseudo-type, `is_iterable` function was introduced in PHP 7.1 and phpunit supports also older versions